lib/jekyll-seo-tag/json_ld_drop.rb

```diff
@@ -36,10 +38,13 @@ def fallback_data
       end
 
       def author
+        author_type = page_drop.author["type"]
+
         return unless page_drop.author["name"]
+        return nil if author_type && !VALID_AUTHOR_TYPES.include?(author_type)
```

Comment:
let us be consistent with the preceding line and return `nil` implicitly:
```suggestion
        return if author_type && !VALID_AUTHOR_TYPES.include?(author_type)
```